linera-rpc/src/simple/client.rs

```diff
@@ -3,24 +3,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::{codec, transport::TransportProtocol};
+
```

Comment:
Nit: same here, following the convention in the rest of the code of just separating imports if they have different visibility modifiers or feature gates.